code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -111,7 +111,7 @@
 	force = 10
 	caliber = "7.62mm"
 	origin_tech = list(TECH_COMBAT = 8, TECH_MATERIAL = 3)
-	ammo_type = "/obj/item/ammo_casing/a556" // Is this really needed anymore?

```

Comment:
I'm not sure why the caliber and ammo_type don't match here. Old issue, probably my fault, will look into it shortly.